desktop-widgets/subsurfacewebservices.cpp

```diff
@@ -55,6 +55,13 @@ static void copy_gps_location(struct dive *from, struct dive *to)
 }
 
 #define SAME_GROUP 6 * 3600 // six hours
+#define SET_LOCATION(_dive, _gpsfix, _mark)\
+{ ;\
+	copy_gps_location(_gpsfix, _dive);\
+	changed ++;\
```

Comment:
@dirkhh , if patches 1, 2 and 4 are finally pushed, would you mind to fix this typo before applying?

Reply:
no problem